deduplicator/src/tests.rs

```diff
@@ -13,6 +13,7 @@ use tempdir::TempDir;
 use crate::deduplicator::Deduplicator;
 
 const DB_NO_DUPES: &str = "data/tests/no_dupes.sql";
+const DB_WITH_DUPES: &str = "data/tests/with_dupes.sql";
```

Comment:
Just realized this is only used in test. It therefore has nothing to do in here.